bitnami/ghost/Chart.yaml

```
@@ -33,4 +33,4 @@ name: ghost
 sources:
   - https://github.com/bitnami/bitnami-docker-ghost
   - https://www.ghost.org/
-version: 16.2.2
+version: 16.2.3
```

Bump the minor version instead since you are adding a new feature.
```suggestion
version: 16.3.0
```